src/NServiceBus.Transport.Msmq/MessagePump.cs

```diff
@@ -134,11 +143,11 @@ async Task ProcessMessages()
             }
         }
 
-        async Task InnerProcessMessages()
+        async Task InnerProcessMessages(CancellationToken cancellationToken)
```

Comment:
would be easier to read if the `cancellationToken`'s naming would be more specific about which token this is. At first glance, it looks this is `messagePumpCancellationTokenSource` anyway and doesn't need to be passed (https://github.com/Particular/NServiceBus.Transport.Msmq/pull/225/files#diff-159e576e4eaa91dfa7405b89fc706862e8655e7c7c84ccc4aee0d5bdc8346098R86)?

Reply:
Agree, using messagePumpCancellationTokenSource  now